react/src/pages/admin/courses/CourseManagementPage.jsx

```
@@ -0,0 +1,79 @@
+            <table className="table table-striped table-hover">
+                <thead className="thead-dark">
+                    <tr>
+                        <th scope="col">#</th>
+                        <th scope="col">Title</th>
+                        <th scope="col">Actions</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {courses.map((course, index) => (
+                        <tr key={course.id}>
+                            <td scope="row">{index + 1}</td>
```

Biome (1.9.4)

[error] Avoid using the scope attribute on elements other than th elements.

The scope attribute is used to associate a data cell with its corresponding header cell in a data table,
            so it should be placed on th elements to provide accessibility to screen readers.
Follow the links for more information,
            WCAG 1.3.1
            WCAG 4.1.1
Unsafe fix: Remove the scope attribute.


(lint/a11y/noHeaderScope)